data/imaaquibali.json

```diff
@@ -55,7 +55,7 @@
     },
     {
       "group": "The VerSiton Community",
-      "name": "EddieHub: Discord",
```

Comment:
I'll get started on a script to add `EddieHub` everywhere. /s 

V small changes, LGTM for what it's worth. 👍

Reply:
Actually i the name of the community is not changed yet and my pervious profile is still visible should i make new pr for that or make changes in this pr

Reply:
This PR is being merged 3 Days ago, so now you need to create a new one for any other changes in your profile.